src/pages/Hub.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from "react";
+import { Container, Row, Col, Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+const Hub = (props) => {
+    const modes = [
+        {
+            name: "Registration",
+            path: "/registration",
+            requiresSession: false
+        },
+        {
+            name: "Login",
+            path: "/login",
+            requiresSession: false
+        },
+        {
+            name: "Logout",
+            path: "/logout",
+            requiresSession: true
+        }
+    ]
+
+    useEffect(() => {
+        props.sessionHandler();
+    });
+
+    return (
+        <Container fluid="lg">
+            <Row>
+                <Col md={{ span: 6, offset: 3 }} className="d-flex align-items-center flex-column">
+                    <h1 className="my-4">Mindtastic Auth API Tester</h1>
+                    {modes.map((item, index) =>
+                        <Link to={item.path} className={"my-4 " + ((item.requiresSession ? !props.session : props.session) ? "disabled-link" : "")} style={{ width: "80%" }} key={index}>
+                            <Button style={{ width: "100%" }} disabled={item.requiresSession ? !props.session : props.session}>
```

Comment:
```suggestion
                        <Link to={item.path} className={"my-4 " + ((item.requiresSession && !userInSession) ? "disabled-link" : "")} style={{ width: "80%" }} key={index}>
                            <Button style={{ width: "100%" }} disabled={item.requiresSession && !userInSession}>
```
We can make the conditionals more expressive by using logic operators.

Reply:
not equivalent to what i want to express here.

Session active --> Registration & Login button disabled
Session inactive --> Logout button disabled

Maybe you have another idea to represent that logic in a more concise way